api/turing/testdata/cluster/servicebuilder/router_configmap_default.yml

```diff
@@ -2,6 +2,7 @@ id: test-svc
 routes:
 - endpoint: http://www.mocky.io/v2/5e4caccc310000e2cad8c071
   id: control
+  protocol: ""
```

Comment:
current FiberConfig has the newly added field `protocol` which is parse to be empty from the api server for now. Subsequently api server will enforce `http` or `grpc`